docs/usecases/anonymizing-sboms/README.md

```diff
@@ -0,0 +1,13 @@
+# Anonymizing SBOMs
+
+## Description
+
+Usecase proposed by @mlieberman85 to "anonymize" an sbom (difference from redact) where a bendor is op with putting an SBOM in a platform like GUAC but something goes wrong, and the vendor wants to send a SBOM with sensitive data anonymized but not scrubbed.
+
+So values like `John Smith <john.smith@acme.com>` becomes `Foo Bar <foo.bar@baz.com>`.
```

Comment:
Oh this is interesting: AFAIK neither one of the SBOM formats support "anonymization" in the sense that both expect a field to be filled or not to be filled (except for SPDX's license NOASSERTION). Wonder if it makes sense to include a "REDACTED" or "ANONYMOUS".